.github/workflows/tests.yml

```diff
@@ -153,7 +153,7 @@ jobs:
       - name: Run test suite
         run: |
           if [[ "${{ matrix.laravel }}" != 'true' && "${{ matrix.coverage }}" = 'true' ]]; then
-            php -d memory_limit=-1 -d zend.enable_gc=0 -d error_reporting=-1 vendor/phpunit/phpunit/phpunit --coverage-clover=coverage.xml --coverage-text;
+            php -d memory_limit=-1 -d zend.enable_gc=0 -d error_reporting=-1 vendor/phpunit/phpunit/phpunit --coverage-clover=clover.xml;
```

Comment:
```suggestion
            php -d memory_limit=-1 -d zend.enable_gc=0 -d error_reporting=-1 vendor/phpunit/phpunit/phpunit --coverage-clover=clover.xml --coverage-text;
```

Reply:
of course, i can change this, and see how the impact on runtime is.
Can you please explain why this is necessary?

Reply:
There is just a little overhead for  `--coverage-text`
Time: 01:35.635, Memory: 1.04 GB

Forget the last question. You want to see the coverage report directly in the github action run.